docs/migrations.rst

```diff
@@ -464,6 +464,38 @@ To simply change the name of the primary key, we need to override the default ``
             }
         }
 
+Valid Table Options
+~~~~~~~~~~~~~~~~~~~~
+
+The following are valid table options:
+
+For all adapters:
+
+=========== ===========
+Option      Description
+=========== ===========
+id          set name of automatically created identity field
+primary_key set field to use for primary key (defaults to value of id)
+=========== ===========
+
+For the MySQL adapter:
+
+========= ===========
+Option    Description
+========= ===========
+collation set collation of table (defaults to utf8_general_ci)
```

Comment:
Looking into [`Phinx\Db\Adapter\MysqlAdapter::createTable`](https://github.com/robmorgan/phinx/blob/0.6.x-dev/src/Phinx/Db/Adapter/MysqlAdapter.php#L243), it doesn't look like charset is supported as a table option (collation is used to determine character set).